docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+        - add a note: `rule_note = "The storage tank is between 2 and 20 gallons.  Appendix F doesn't provide an efficiency for this water heaters of this size."
+    - otherwise, continue with rule logic: `else:`
+      - check electric storage water heaters with capacity greater than 12 kW: `if swh_type == "ELECTRIC_RESISTANCE_STORAGE" && input_capacity > 40945.7:`
+        - check that the capacity_per_volume < 4000 btu/hr/gallon: `if capacity_per_volume < 4000:`
+          - calculate the standby loss target: `standby_loss_target = 0.3 + (27/swh_eq.distribution_system.tank.storage_volume)`
+          - 
+          - if the standby_loss is less than or equal to the target, set the rule_status to PASS: `if standby_loss <= standby_loss_target: rule_status: "PASS"`
+          - otherwise: `else:`
+            - set the note to indicate that the standby_loss is higher than the maximum allowed: `rule_note = "Standby Loss is higher than required by Table 7.8"`
+        - otherwise: `else:`
+          - set the note to indicate that the capacity per volume exceeds the limit: `rule_note = "Capacity per volume exceeds the limit of 4000 btu/hr/gallon for Electric Storage Water Heaters` 
+      - check if the water heater is gas storage: `if swh_type == "GAS_STORAGE":`
+        - check if the capacity is <= 75,000 btu/hr: `if input_capacity <= 75000:`
+          - if the storage volume is between 20 and 55 gallons: `if storage_volume >= 20 && storage_volume <= 55:`
+            - determine the UEF based on the draw pattern for VERY_SMALL: `if draw_pattern == "VERY_SMALL": expected_uef = 0.3456 – (0.0020 × storage_volume)`
+            - determine the UEF based on the draw pattern for LOW: `if draw_pattern == "LOW": expected_uef = 0.5982 – (0.0019 × storage_volume)`
+            - determine the UEF based on the draw pattern for MEDIUM: `if draw_pattern == "MEDIUM": expected_uef = 0.6483 – (0.0017 × storage_volume)`
+            - determine the UEF based on the draw pattern for HIGH: `if draw_pattern == "HIGH": expected_uef = 0.6920 – (0.0013 × storage_volume)`
+            - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+              - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF:`
+                - set rule_status to "PASS": `rule_status = "PASS"`
+              - otherwise, set rule status to FAIL and provide the note that the expected UEF is less than the required UEF: `else: rule_status = "FAIL"; rule_note = "The modeled Uniform energy Factor (UEF - " + uniform_energy_factor + ") is less than required (" + expected_UEF + ")"`
+            - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+          - otherwise, if the storage volume is between 55 and 100 gallons: `elif storage_volume <= 100:`
```

Comment:
Suggest adding a case that captures when the storage volume is less than 20 gal including a note something like "The storage tank volume is less than 20 gallons and based on the size categories in ASHRAE 90.1 Table 7.8 and Appendix F it would not be classified as a gas storage water heater. Consequently, this rule was not able to be assessed for this service water heater. "